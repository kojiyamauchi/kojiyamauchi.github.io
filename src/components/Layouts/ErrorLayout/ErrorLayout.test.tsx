/**
 * @jest-environment jsdom
 */
/*
  Error Layout Component Unit Test.
*/

import { ErrorLayout } from './'
import { render /* ,screen */ } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
// import userEvent from '@testing-library/user-event'

describe('Fixed Layout Component Unit Test', () => {
  it('Snap Shot Testing', () => {
    const { asFragment } = render(<ErrorLayout pages="ui-test" />)
    expect(asFragment()).toMatchSnapshot()
  })
})
