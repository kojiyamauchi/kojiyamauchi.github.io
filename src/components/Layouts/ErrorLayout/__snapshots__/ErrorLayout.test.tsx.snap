// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fixed Layout Component Unit Test Snap Shot Testing 1`] = `
<DocumentFragment>
  <div
    class="ui-test wrapper error-layout"
  >
    <header
      class="wrapper error-page-header-container"
    >
      <h1
        class="primary-heading error-page-primary-heading"
      >
        Error Page.
      </h1>
      <ul
        class="wrapper error-page-navigation"
      >
        <li
          class="error-page-navi-list"
        >
          <a
            class="link error-page-navi-link"
            href="/"
          >
            Back to Top Page
          </a>
        </li>
      </ul>
    </header>
    <main
      class="main-content main-content"
      style="background-image: url(webps/logo.webp);"
    />
    <footer
      class="wrapper error-page-footer-container"
    >
      <a
        class="wrapper undefined is-error"
        href="http://kojiyamauchi.com"
        rel="noreferrer"
        target="_blank"
      >
        Â© 2022 All The Small Things, All Rights Reserved.
      </a>
    </footer>
  </div>
</DocumentFragment>
`;
